LibreNMS/Util/Rewrite.php

```diff
@@ -135,6 +135,17 @@ public static function shortenIfName($name)
         return str_ireplace(array_keys($rewrite_shortif), array_values($rewrite_shortif), $name);
     }
 
+    /**
+     * Remove all allowed delmiters and convert to lower
+     *
+     * @param string $mac
+     * @return string
+     */
+    public static function normalizeMac($mac)
```

Comment:
Indeed as Jellyfrog has mentioned: Rewrite::macToHex()  is the same as this, but more robust (handles macs omitting leading zeros)

Reply:
Ups, I haven't seen it. It's much more better then my method. I will convert it to Rewrite::macToHex()